specs/protocol/derivation.md

```diff
@@ -331,7 +331,7 @@ Batcher transactions are encoded as `version_byte ++ rollup_payload` (where `++`
 | `version_byte` | `rollup_payload`                               |
 | -------------- | ---------------------------------------------- |
 | 0              | `frame ...` (one or more frames, concatenated) |
-| 1    | `plasma_commitment` (experimental, see [op-plasma](../experimental/plasma.md#input-commitment-submission) |
+| 1    | `plasma_commitment` (experimental, see [alt-da](../experimental/alt-da.md#input-commitment-submission)) |
```

Comment:
I guess we don't want to rename this to `da_commitment` in this PR?

Reply:
Oh good catch, I'll do that one too.